packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -67,14 +67,15 @@ describe('RPC Server Acceptance Tests', function () {
     const FEE_SCHEDULE_FILE_ID = "0.0.111";
     const FEE_SCHEDULE_FILE_CONTENT_DEFAULT = "0a280a0a08541a061a04408888340a0a08061a061a0440889d2d0a0a08071a061a0440b0b63c120208011200"; // Eth gas = 853000
     const FEE_SCHEDULE_FILE_CONTENT_UPDATED = "0a280a0a08541a061a0440a8953a0a0a08061a061a0440889d2d0a0a08071a061a0440b0b63c120208011200"; // Eth gas = 953000
+    const publicNetworks = ['previewnet', 'testnet', 'mainnet'];
 
     describe('RPC Server Acceptance Tests', function () {
         this.timeout(240 * 1000); // 240 seconds
 
         this.beforeAll(async () => {
-            accounts[0] = await servicesNode.createAliasAccount();
-            accounts[1] = await servicesNode.createAliasAccount();
-            accounts[2] = await servicesNode.createAliasAccount(20);
+            accounts[0] = await servicesNode.createAliasAccount(20);
+            accounts[1] = await servicesNode.createAliasAccount(20);
+            accounts[2] = await servicesNode.createAliasAccount(50);
```

Comment:
Q: what's governing the extra 30 hbar increase?
Are there any entities like tokens or contracts etc that can be shared to minimize create cost?

Reply:
I optimized the hbars. The increase was due to the higher gas price on the public nets.

Reply:
Gotcha